src/languageservice/services/yamlHover.ts

```diff
@@ -86,9 +88,10 @@ export class YAMLHover {
     );
 
     const createHover = (contents: string): Hover => {
+      const regex = new RegExp(this.indentation, 'g');
```

Comment:
@rgrunber 
Tabs and whitespace have a special meaning in [Markdown.](https://www.markdownguide.org/hacks/#indent-tab) So converted spaces into `non braking space` as a workaround here.